cglicenses.json

```diff
@@ -530,5 +530,11 @@
 			"IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN",
 			"CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE."
 		]
+	},
+	{
+		"name": "cacheable-request",
+		"prependLicenseText": [
+			"Copyright (c) cacheable-request authors"
```

Comment:
Isn't it using an MIT license? https://github.com/jaredwray/cacheable/blob/main/LICENSE

Reply:
This is because there is no copyright statement in https://github.com/jaredwray/cacheable/blob/main/LICENSE. It is like https://github.com/microsoft/vscode/blob/6c9d42afa8913eb6e3cd2f0d879b2ffd069722db/cglicenses.json#L32